oauth2server/proj/settings/cloud_foundry_settings.py

```diff
@@ -91,7 +91,7 @@
     }
 }
 
-DEBUG = True                                                # Set to false on Production
+DEBUG = os.environ.get('DEBUG','False')
```

Comment:
Does this work? `os.environ.get` returns a string, and the default is a string, rather than the expected boolean. 

`strtobool` can be used to convert the strings to a boolean (see https://github.com/ONSdigital/ras-frontstage/commit/7bb554bce7bcc26c92f404c857d6245ebead6ec3)

Reply:
fixed